src/lib.rs

```diff
@@ -710,6 +712,11 @@ impl Jieba {
             .collect();
         tags
     }
+
+    pub fn tfidf<'a>(&'a self, sentence: &'a str, top_k: usize, allowed_pos: Vec<String>) -> Vec<String> {
+        let keyword_extractor = tfidf::TFIDF::new();
```

Comment:
`TFIDF::new` will load TFIDF dict every time, maybe we should make it a `lazy_static`?

Reply:
On a second thought. I'll move this method outside of `Jieba`, that is to be a public function tfidf in lib.rs but not a method of Jieba, and lazy_static a DEFAULT_TFIDF_KEYWORD_EXTRACTOR to hold the instance.